packages/sitecore-jss-nextjs/src/middleware/healthcheck-middleware.ts

```diff
@@ -0,0 +1,18 @@
+import { NextApiResponse, NextApiRequest } from 'next';
```

Comment:
Hmm.. don't know that `middleware` is the appropriate place for this either. These should only be middlewares used on the [Edge runtime](https://nextjs.org/docs/api-reference/edge-runtime). We might be OK now since this `HealthcheckMiddleware` isn't using any native Node.js APIs, but once it did it would likely cause build errors for the other middlewares here.

Guess we're talking about a new submodule then??

Reply:
It's a valid point for the future, I added `monitoring` submodule